Project.toml

```diff
@@ -11,6 +11,7 @@ HistogramThresholding = "2c695a8d-9458-5d45-9878-1b8a99cf7853"
 ImageContrastAdjustment = "f332f351-ec65-5f6a-b3d1-319c6670881a"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MappedArrays = "dbb5928d-eab1-5f90-85c2-b9b0edb7c900"
```

Comment:
Where do we use this package?

Reply:
`of_eltype(Gray, img)` performs almost the same as `Gray.(img)` except that it doesn't allocate additional memory.

```julia
julia> @benchmark of_eltype(Gray, img)
BenchmarkTools.Trial: 
  memory estimate:  16 bytes
  allocs estimate:  1
  --------------
  minimum time:     29.858 ns (0.00% GC)
  median time:      31.279 ns (0.00% GC)
  mean time:        37.766 ns (14.93% GC)
  maximum time:     40.667 μs (99.91% GC)
  --------------
  samples:          10000
  evals/sample:     994

julia> @benchmark Gray.(img)
BenchmarkTools.Trial: 
  memory estimate:  508.19 KiB
  allocs estimate:  4
  --------------
  minimum time:     31.353 μs (0.00% GC)
  median time:      38.761 μs (0.00% GC)
  mean time:        50.567 μs (22.40% GC)
  maximum time:     42.538 ms (99.85% GC)
  --------------
  samples:          10000
  evals/sample:     1
```